packages/babel-plugin/src/utils/css-builders.ts

```diff
@@ -701,9 +635,12 @@ const extractTemplateLiteral = (node: t.TemplateLiteral, meta: Metadata): CSSOut
     // E.g. `font-size: ${fontSize}px` will end up needing to look like:
     // `font-size: var(--_font-size)`, with the suffix moved to inline styles
     // style={{ '--_font-size': fontSize + 'px' }}
-    const name = `--_${hash(variableName)}`;
     const nextQuasis = node.quasis[index + 1];
     const [before, after] = cssAffixInterpolation(quasi.value.raw, nextQuasis.value.raw);
+    // Create a different CSS var name for negative value version
+    const name = `--_${hash(variableName)}${
```

Comment:
Replaced the negative number variable fix with this. Basically generate a different name for the negative version of the interpolation.